sdk/tracing/azure-core-tracing-opencensus/README.md

```diff
@@ -0,0 +1,223 @@
+        .endpoint("<your-vault-url>")
+        .credential(new DefaultAzureCredentialBuilder().build())
+        .buildClient();
+    
+    Tracer tracer = Tracing.getTracer();
+    
+    Scope scope = tracer.spanBuilder("tracing-user-span").startScopedSpan())    
+    // Set secret and pass tracing context with method call
+    secretAsyncClient.setSecret(new Secret("secret_name", "secret_value")
+                    .subscriberContext(Context.of(OPENCENSUS_SPAN_KEY, tracer.getCurrentSpan()).doFinally(secret ->
+                        System.out.printf("Secret is created with name %s and value %s %n", secret.getName(), secret.getValue()));
+    ```
+
+### Using the plugin package with AMQP client libraries
+Async send single event using [azure-messaging-eventhubs][azure-messaging-eventhubs] with tracing.
+    
+Users can additionally pass the value of the current tracing span to the EventData object with key **"opencensus-span"** on the [Context][context] object:
+
+```java
+import io.opencensus.common.Scope;
+import io.opencensus.trace.Tracer;
+import io.opencensus.trace.Tracing;
+import com.azure.core.util.Context;
+import com.azure.messaging.eventhubs.EventData;
+import com.azure.messaging.eventhubs.EventHubAsyncClient;
+import com.azure.messaging.eventhubs.EventHubAsyncProducer;
+import com.azure.messaging.eventhubs.EventHubClientBuilder;
+
+EventHubAsyncClient client = new EventHubClientBuilder()
+    .connectionString("<< CONNECTION STRING FOR SPECIFIC EVENT HUB INSTANCE >>")
+    .buildAsyncClient();
+EventHubAsyncProducer producer = client.createProducer();
+
+Scope scope = tracer.spanBuilder("tracing-user-span").startScopedSpan())
+Context tracingContext = new Context(OPENCENSUS_SPAN_KEY, tracer.getCurrentSpan());
+// Create an event to send
+final EventData eventData = new EventData("Hello world!".getBytes(UTF_8));
+// Add tracing context to the event
+eventData.context(tracingContext);
+producer.send(eventData).doFinally((ignored) -> System.out.println("Event sent."));
+```
+
+## Troubleshooting
+### General
+
+For more information on Opencensus Java support for tracing, see [Opencensus Java Quickstart][opencensus-quickstart].
+Please refer to the [Quickstart Zipkin][zipkin-quickstart] for more documentation on using a zipkin exporter.
+
+## Next steps
+Several Java SDK samples are available to you in the SDK's GitHub repository. These following samples provide example code for additional scenarios commonly encountered while working with Tracing:
+
+#### Enqueue and Dequeue messages using Storage Queue client samples
+* [HelloWorld.java][sample_helloWorld] and [HelloWorldAsync.java][sample_helloWorldAsync] - Contains samples with tracing enabled for following scenarios:
+    * Create a Queue client using [azure-storage-queue][azure-storage-queue] Java SDK.
+    * Enqueue and Dequeue messages using the created Queue client.
+#### List Operations for Secrets in a Keyvault samples
```

Comment:
List operations for secrets in a Key Vault

Reply:
I don't think secrets should be capitalised. In the Azure documentation, it is sentence cased properly.

https://docs.microsoft.com/en-us/azure/key-vault/

Reply:
The product name is "Key Vault"